Assets/Scripts/Utilities/MathUtils.cs

```diff
@@ -85,4 +85,10 @@ public static bool RandBool()
 	}
 
 	#endregion
+
+	public static Color SetAlpha( this Color color, float alpha )
```

Comment:
I think a cleaner way to do this would be ColorUtils.SetAlpha( ref Color color, float alpha). Then using it in code doesn't require the weird readability of color = color.SetAlpha(blah)